### ISSUE TITLE
The chart's minimum width is 300px when it's set to 100%

### ISSUE THREAD

**Issue by hafezdivandari** [chartist-js/chartist#1262]
The chart's width doesn't go below 300px when its width option is set to 100% on chrome: https://jsfiddle.net/hafez/9w5up1b4/

![image](https://user-images.githubusercontent.com/56585913/89116254-57f89500-d481-11ea-86a3-eb70856a7bb2.png)

**Comment by Arantiryo** [comment]
Hi @hafezdivandari 

Is the issue still relevant for you? Looks like it works fine with the latest version.

**Comment by hafezdivandari** [comment]
Hi @Arantiryo, It has been fixed on the latest version, thanks.

**Comment by hafezdivandari** [comment]
Hi @Arantiryo 

I tested the latest version and updated the fiddle https://jsfiddle.net/hafez/9w5up1b4/

As you can see on the fiddle, the issue is not fixed yet.

**Comment by Arantiryo** [comment]
Looks like using `min-width` instead of `width` is the issue. 

Setting `style="width: 200px;"` for `container` div makes the chart 200px wide.

**Comment by hafezdivandari** [comment]
@Arantiryo yeah I know that as a workaround, but I have a complex use case and I've chosen chartist for it's responsive feature. But it seems that 200px width is hardcoded somewhere / somehow in the chartist that is breaking this.
